portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmailReport.testcase

```diff
@@ -41,6 +41,72 @@ definition {
 		}
 	}
 
+	@description = "Story: LRAC-11664 | Automation ID: LRAC-11841 | Test Summary: Can change email report frequency to daily, weekly or monthly"
+	@priority = "3"
+	test CanChangeEmailReportFrequency {
+		var frequencyList = "Daily,Weekly,Monthly";
+
+		task ("Go to AC > Go to Settings > Go to Properties > Open the new property") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			ACSettings.goToProperties();
+
+			ACNavigation.openItem(itemName = "${assignedPropertyName}");
+		}
+
+		task ("Click on the email report widget and enable it and then save the report to: daily, weekly and monthly") {
+			for (var frequencyOption : list "${frequencyList}") {
+				ACEmailReport.enableEmailReport(frequency = "${frequencyOption}");
+
+				ACUtils.viewGenericText(textValueList = "Email Reports: Enabled");
+
+				Alert.viewSuccessMessageText(successMessage = "Changes to email reports saved.");
+
+				ACEmailReport.disableEmailReport();
+			}
+		}
+	}
+
+	@description = "Story: LRAC-11664 | Automation ID: LRAC-11846 | Test Summary: Disable email report after enabling"
+	@priority = "3"
+	test CanDisableEmailReportAfterEnabling {
+		task ("Go to AC > Go to Settings > Go to Properties > Open the new property") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			ACSettings.goToProperties();
+
+			ACNavigation.openItem(itemName = "${assignedPropertyName}");
+		}
+
+		task ("Click the widget next to email reports and then enable it") {
+			ACEmailReport.accessEmailReportSettings();
+
+			ACEmailReport.enableEmailReportToggle();
+
+			Button.clickSave();
```

Comment:
Hey @manuferreira, I think your steps are correct, but I was wondering if you could use `ACEmailReport.enableEmailReport` and perform these 3 lines of code in 1 line only. What do you think?

Reply:
I was wondering about this too, this macro can just use solely for enabling Email Report. And maybe one just for disabling. On line 212 - 216, we can use the disabling macro too. What do you think?

Reply:
Hi @ViniLopes20, thanks for noticing this and for the suggestion.

Hi @calvinctvo, thanks for the suggestion. About lines 212 - 216, I can't use the disabling macro because I have this code in line 204:
```
ACEmailReport.accessEmailReportSettings();
```
I need to access the email report settings and assert if the frequency is still daily. In lines 212 - 216 I didn't use the `disableEmailReport` because I already have this step `ACEmailReport.accessEmailReportSettings();` and I'm already inside the report settings. So for me, it wouldn't make sense to use the disable macro and repeat the step of accessing the email report settings. So what do you think?

Reply:
Everything said here makes sense to me. It's a small thing, but as Vini said, the test can be made a little cleaner with `ACEmailReport.enableEmailReport`

Otherwise, everything is looking good to me!